### PR TITLE
Allow setup of a question without a correct answer

### PR DESCRIPTION
**Pull request by jleong-openedx**
An instructor can now specify that a PI question has no correct answer.
The statistics graphs do not indicate a correct answer if none was
specified.

The student's response is also no longer indicated as incorrect (ie
rendered in red) for this case.

### CODE REVIEW

**Comment by jleong-openedx** [comment]
Closes #90.

File: ubcpi/answer_pool.py [hunk 1]
```diff
@@ -119,9 +119,10 @@ def validate_seeded_answers_simple(answers, options, algo):
     index = 1
     for option in options:
         key = option.get('text') + option.get('image_url') if option.get('image_url') else option.get('text')
-        if seen_options.get(key, 0) == 0:
-            missing_options.append('Option ' + str(index))
-        index += 1
+        if option.get('text')!='No correct answer':
```
Comment (xcompass) [inline ubcpi/answer_pool.py]: NIT: could you add spaces before and after !=

File: ubcpi/static/js/src/ubcpi_edit.js [hunk 2]
```diff
@@ -131,6 +145,8 @@ angular.module("ubcpi_edit", ['ngMessages', 'ngSanitize', 'ngCookies'])
 
             self.submit = function() {
                 notify('save', {state: 'start', message: "Saving"});
+                if(data.correct_answer==data.options.length)
```
Comment (xcompass) [inline ubcpi/static/js/src/ubcpi_edit.js]: NIT: adding spaces after if and before and after ==

Comment (jleong-openedx) [inline ubcpi/static/js/src/ubcpi_edit.js]: Okay done, I have amended this.

File: ubcpi/answer_pool.py (hunk 1 above)
Comment (jleong-openedx) [inline ubcpi/answer_pool.py]: Amended.

**Pull request merged**